guidelines/templates/resources/release-notes.md

```diff
@@ -51,10 +54,14 @@ redirectFrom:
 
 > Write a short paragraph that describes the feature or the fix in details and explains its benefits to the Kyma users. Include screenshots to illustrate the change better.
 
-### Known issues
+## Known issues
 
 > Describe any known issues that the users can face, together with the way on how to solve these issues.
 
-### Fixed security vulnerabilities
+## Migrations and upgrades
 
-> Describe the solved security vulnerability issues related to the Kyma project. Provide a short issue description, its calculated risk assessment, and a link to the pull request that solves the issue. You can also include a GitHub link to the issue itself. The calculated risk assessment is provided in each issue of the [Security Vulnerability](https://github.com/kyma-project/kyma/issues/new?template=security-vulnerability.md) type created on Github.
+> Describe any component-related migrations or upgrades and reasons behind them. Link to a separate migration guide under `kyma/docs/migration-guides` for manual steps the users must perform to migrate to a new Kyma version successfully.
```

Comment:
Maybe we should add info about the form of this section? Should it be a list with components and descriptions of their migrations, or rather full paragraphs with no sub-sections?

I think this would be more clear:
```
### {Component name}
> Describe changes here.
```

I have the same concerns about the `Known issues` section, now that it is brought out from under the components sections.